test/test_customs_on_service_hosgroups.py

```diff
@@ -47,38 +47,40 @@
 # This file is used to test reading and processing of config files
 #
 
-from alignak_test import *
+import unittest
+from alignak_test import AlignakTest
 
 
 class TestCustomsonservicehosgroups(AlignakTest):
 
     def setUp(self):
-        self.setup_with_file(['etc/alignak_customs_on_service_hosgroups.cfg'])
+        self.setup_with_file('cfg/cfg_customs_on_service_hosgroups.cfg')
+        self._sched = self.schedulers['scheduler-master'].sched
 
     # We look for 3 services: on defined as direct on 1 hosts, on other
     # on 2 hsots, and a last one on a hostgroup
     def test_check_for_custom_copy_on_serice_hostgroups(self):
```

Comment:
Missing test description. Add a 
""" We look for 3 services: on defined as direct on 1 hosts, on other on 2 hosts, and a last one on a hostgroup
"""

Same comment for all the tests :wink: